guides/common/modules/proc_viewing-audit-logging-information.adoc

```diff
@@ -0,0 +1,11 @@
+[id="viewing-audit-logging-information_{context}"]
+= Viewing Audit Logging Information
+
+Audit records list the changes made by users on the {Project} server.
+In the  {ProjectWebUI}, navigate to *Monitor* > *Audits* to view the audit records.
+
+To obtain a list of all the audit attributes, use the following command:
```

Comment:
Hey @maximiliankolb I just realized that this command isn't exactly the CLI procedure to list the audit logging info, it only lists all the audit attributes and their definitions. Do you think it would be best to have a single line procedure and then a follow-up line outside of the procedure that introduces the command (the same content that we have now placed under `CLI procedure`)?